File: typescript/examples/hello-world.ts
import { DriftGateClient } from "../src/index.ts";

const dg = new DriftGateClient({ baseUrl: "https://api.driftgate.ai", sessionToken: "token" });
const session = await dg.session.start({ agent: "refund-agent" });
await session.execute({ input: { orderId: "123" } });

File: ruby/examples/hello_world.rb
# frozen_string_literal: true

require_relative "../lib/driftgate_sdk"

client = DriftGateSDK::Client.new(base_url: "https://api.driftgate.ai")
session = client.session.start(agent: "refund-agent")
session.execute(input: { orderId: "123" })
